com.unity.render-pipelines.universal/Runtime/UniversalRenderPipeline.cs

```diff
@@ -174,7 +174,9 @@ public static void RenderSingleCamera(ScriptableRenderContext context, Camera ca
             if (asset.useAdaptivePerformance)
                 ApplyAdaptivePerformance(ref cameraData);
 #endif
-            RenderSingleCamera(context, cameraData, cameraData.postProcessEnabled);
+
+            bool anyPostProcessing = cameraData.postProcessEnabled && asset.postProcessIncluded;
```

Comment:
I can totally see RenderingSingleCamera(ctx, cam, bool, bool, bool, bool, bool, bool, bool, ... ) being thing in the future. Should we have a settings/options struct, pass in the asset and ask from it or at least do an enum i.e. that would read like PostProcess.Enabled/.Disabled?

Reply:
Makes perfect sense. I would like to do that in a separate PR though.